hazelcast/src/main/java/com/hazelcast/query/PagingPredicateAccessor.java

```diff
@@ -16,6 +16,8 @@
 
 package com.hazelcast.query;
 
+import com.hazelcast.query.impl.predicates.PagingPredicateImpl;
+
 import java.util.Map;
 
```

Comment:
I think we should add a javadoc to `PagingPredicateAccessor` saying something like "this class is for internal use only and may be changed without any notice".

Reply:
Good idea, will do.